fbcp-ili9341.cpp

```diff
@@ -273,6 +277,7 @@ int main()
       __atomic_fetch_sub(&numNewGpuFrames, numNewFrames, __ATOMIC_SEQ_CST);
 
       DrawStatisticsOverlay(framebuffer[0]);
+      drawLowBatteryIcon(framebuffer[0]);
```

Comment:
Can you check the case of the function names (variables are lowerCamelCase, functions are UpperCamelCase)?